content/pages/cloud.json

```
@@ -35,7 +35,7 @@
           "variant": "orange",
           "label": "Sign up",
           "icon": "arrowRight",
-          "url": "https://auth.tina.io/register"
+          "url": "http://app.tina.io/register"
```

```suggestion
          "url": "https://app.tina.io/register"
```
(it should redirect anyway)